lib/ukrust/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+license-file = "../../LICENSE.txt"
+name = "ukrust"
+version = "0.1.0"
+edition = "2018"
```

Comment:
This should be updated to 2021 right?

Reply:
Yeah, I did try that initially, but it doesn't build because the core crate is built using the 2018 edition[1].  Though, in the future we should definitely update it. Thanks Dennis!

[1] https://github.com/unikraft/unikraft/blob/staging/lib/ukrust/Makefile.uk#L19

Reply:
Ah, that makes sense